arch/x86/boot/compressed/kaslr.c

```diff
@@ -302,6 +307,14 @@ static void handle_mem_options(void)
 		} else if (!strcmp(param, "efi_fake_mem")) {
 			mem_avoid_memmap(PARSE_EFI, val);
 		}
+#ifdef CONFIG_SECURITY_TEMPESTA
+		else if (!strcmp(param, "tempesta_dbmem")) {
+			char *p = val;
+
+			tempesta_dbmem_sz = round_up(memparse(p, &p),
```

Comment:
Tempesta is 32MB per node. Default is 512MB.
I think we need to check it and set appropriately like it is done for !strcmp(param, "mem")
```
x = memparse(p, &p),
if x < 32MB
    x =32MB
```

Reply:
I'm not sure, that we should try to clamp the region if `tempesta_dbmem` below than 512MB or even 1GB